app/models/reservation.rb

```diff
@@ -0,0 +1,12 @@
+class Reservation < ApplicationRecord
+  belongs_to :user
+  belongs_to :event
+  belongs_to :skill
+  belongs_to :reservation_status
+  has_many :driven_pairs, class_name: 'Pair', foreign_key: 'person_1'
+  has_many :navigated_pairs, class_name: 'Pair', foreign_key: 'person_2'
+
+  def pairs
+    driven_pairs + navigated_pairs
```

Comment:
Ok, I think I understand. It looks like the Reservation model allows a user to update their skill and experience levels per event.

I think keeping track of driver and navigator would be good too. Would it make more sense to have `User` know about these, instead of `Reservation`? 

e.g. `@user.driven_pairs`

I think Reservation belonging to User is something I'm still confused about. I was thinking they would belong to Event.